grp/perf34.grp

```diff
@@ -0,0 +1,158 @@
+#############################################################################
+##
+##  This file is part of GAP, a system for computational discrete algebra.
+##  It contains the perfect groups of order 1975680
+##  This data was computed by Alexander Hulpke
+##  It is distributed under the artistic license 2.0
+##  https://opensource.org/licenses/Artistic-2.0
```

Comment:
Hmmm, that's a problem from a legal point of view. Right now we claim "GAP is GPL 2", period, without any differentation. And that's correct at least for core GAP (not for package). This PR changes that. This will require some extra changes. I can help with those, but this is a blocker for a release, IMHO.